packages/ssx-authjs/src/client/svelte.ts

```diff
@@ -0,0 +1,34 @@
+import { signIn } from '@auth/sveltekit/client';
```

Comment:
```suggestion
```
This import command without the "type: module" on the package.json triggered a TS1479 error. 
As suggested by the error message "Consider writing a dynamic 'import("@auth/sveltekit/client")'".
So we will remove this import and update line 16